src/components/DNSMap/DNSMap.tsx

```diff
@@ -30,14 +19,39 @@ interface FormValuesType {
     ipsToIgnore: string;
 }
 
-const DNSMap = () => {
-    const [loading, setLoading] = useState(false);
-    const [output, setOutput] = useState("");
-    const [checkedAdvanced, setCheckedAdvanced] = useState(false);
-    const [Pid, setPid] = useState("");
-    const [allowSave, setAllowSave] = useState(false);
-    const [hasSaved, setHasSaved] = useState(false);
+/**
+ * The DNSMap component.
+ * @returns The DNSMap component.
+ */
 
+const DNSMap = () => {
+    //Componenets state Variable
+    const [loading, setLoading] = useState(false); // State variable that represents loading state of the component
+    const [output, setOutput] = useState(""); // State variable that represents the output text from the DNS mapping process
+    const [checkedAdvanced, setCheckedAdvanced] = useState(false); //State variable that represents the state of advanced mode switch
+    const [Pid, setPid] = useState(""); //State variable that represents the process ID of the DNS mapping process
+    const [allowSave, setAllowSave] = useState(false); //State variable that represents whether saving the output is allowed
+    const [hasSaved, setHasSaved] = useState(false); //State variable that represents whether the output has been saved
+
+    //Components Constant Variables
+    const title = "DNS Mapping for Subdomains (DNSMap)";
```

Comment:
Need to match the title with what's on Kali.org, so it should be Dnsmap. 
```suggestion
    const title = "Dnsmap";
```